etc/settings.cfg.sh

```diff
@@ -25,9 +25,7 @@ export EUPSPKG_SOURCE=${EUPSPKG_SOURCE:-git}
 export EUPS_PKGROOT=$LSSTSW/distserver/production
 
 # the location of source repositories
-BASE='git://git.lsstcorp.org/LSST'
-SBASE='https://stash.lsstcorp.org/scm'
-export REPOSITORY_PATTERN="$BASE/sims/%(product)s.git|$BASE/DMS/%(product)s.git|$BASE/DMS/devenv/%(product)s.git|$BASE/DMS/testdata/%(product)s.git|$BASE/external/%(product)s.git|$SBASE/sim/%(product)s.git"
+export REPOFILE=${LSSTSW}/etc/repos.yaml
```

Comment:
The point is *not* to have every repository explicitly mapped -- only those that can't fit the pattern  After we move, I'm guessing we'll have only a couple of entries in the REPOSITORY_PATTERN.  It's much easier administratively not to have to update a file (and all users of that file) every time a product is added.  Since we'll only have a couple of potential locations, and since it will be a huge problem to have duplicate names anyway, I don't think the "creating a new repository at a preceding path" is really much of a problem.